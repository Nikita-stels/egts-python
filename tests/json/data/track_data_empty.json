{
  "transport": {
    "pt": 1
  },
  "service": {
    "sdr": [
      {
        "rfl": {
          "ssod": 0,
          "rsod": 0
        },
        "tm": "0x0fd90000",
        "sst": 10,
        "rst": 10,
        "rd": [
          {
            "srt": 62,
            "srd": {
              "atm": 300000
            }
          }
        ]
      }
    ]
  }
}
