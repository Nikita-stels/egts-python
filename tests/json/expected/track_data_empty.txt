EGTS protocol validator v.1.0  Copyright JSC "Navigation-information systems", 2011
------------------------------------------------------------------------------------
File: 'D:\PycharmProjects\egts_protocol\tests\tmp_msg.bin'
Packet data: 
 0100000B0014000100010009000100040000D90F0A0A3E060001E09304000022C4


EGTS Transport Layer:
---------------------
  Validating result   - 0 (OK)

  Protocol Version    - 1
  Security Key ID     - 0
  Flags               - 00000000b (0x00)
       Prefix         - 00
       Route          -   0
       Encryption Alg -    00
       Compression    -      0
       Priority       -       00 (the highest)
  Header Length       - 11
  Header Encoding     - 0
  Frame Data Length   - 20
  Packet ID           - 1
  No route info       -
  Header Check Sum    - 0x00

EGTS Service Layer:
---------------------
  Validating result   - 0 (OK)

  Packet Type         - EGTS_PT_APPDATA
  Service Layer CS    - 0xC422

    Service Layer Record:
    ---------------------
    Validating Result    - 0 (OK)

    Record Length               - 9
    Record Number               - 1
    Record flags                -     00000100b (0x04)
        Sourse Service On Device    - 0
        Recipient Service On Device -  0
        Group Flag                  -   0
        Record Processing Priority  -    00 (the highest)
        Time Field Exists           -      1
        Event ID Field Exists       -       0
        Object ID Field Exists      -        0
    Time                        - 0x0FD90000 (05.06.2018 07:25:52 GMT)
    Source Service Type         - 10 (EGTS_ECALL_SERVICE) 
    Recipient Service Type      - 10 (EGTS_ECALL_SERVICE) 

       Subrecord Data:
       ------------------
       Validating Result   - 0 (OK)

       Subrecord Type      - 62 (EGTS_SR_ECALL_TRACK_DATA)
       Subrecord Length    - 6
       Structures Amount      - 1
       Absolute Time          - 0x000493E0 (04.01.2010 11:20:00.000  GMT)
       Track Points Data Structures:
       -----------------------------


-------------------------------------------------------------
Press any key to exit...
